linter/rules/label_custom_check.py

```diff
@@ -11,12 +11,16 @@ def applies_to() -> Tuple[str, ...]:
     def run(self, lookml_object, runtime_params: Union[Any, None] = None) -> bool:
         label = lookml_object.get('label')
         if label:
-            words_in_label = label.split(' ')
-            for word in words_in_label:
-                valid = starts_with_capital_or_digit_or_special_char(word)
-                if not valid:
-                    return False
+            # Check if all characters are whitespaces
+            if all(char.isspace() for char in label):
+                return False  # Consider it invalid if all characters are whitespaces
+
+            # Check if the remaining characters follow the rule
+            words_in_label = label.split()
+            return all(starts_with_capital_or_digit_or_special_char(word) for word in words_in_label)
+
         return True
 
+
     def message(self) -> str:
-        return 'Every word in label should start with capital letter, digit or special character'
+        return 'Every word in label should start with capital letter, digit, special character or whitespace'
```

Comment:
why word should start with whitespace?

Reply:
I guess better to highlight that label cannot be blank and should start with ...

Reply:
Ah, thank you for catching that! Should be good now ✅